build/params_mainnet.go

```diff
@@ -169,6 +170,10 @@ const Eip155ChainId = 314
 // WhitelistedBlock skips checks on message validity in this block to sidestep the zero-bls signature
 var WhitelistedBlock = MustParseCid("bafy2bzaceapyg2uyzk7vueh3xccxkuwbz3nxewjyguoxvhx77malc2lzn2ybi")
 
-const f3Enabled = false
-const ManifestServerID = "12D3KooWENMwUF9YxvQxar7uBWJtZkA6amvK4xWmKXfSiHUo2Qq7"
-const F3BootstrapEpoch abi.ChainEpoch = -1
+// TODO: We need F3Enabled to be public because without it there is no way to
+// allow f3 pubsub topics without a lot of refactoring.
+var F3Enabled = false
```

Comment:
Unfortunately, this one needs to be public so it can be enabled at will in itests. Without them, pubsub F3 topics won't be allowed. For the rest we don't need it because we manually build the node F3-enabled through DI.